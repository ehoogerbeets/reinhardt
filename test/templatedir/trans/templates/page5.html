{% trans "Source string." as foobar %}
<div>
   Untranslated string with {{ foobar }} in the middle.
</div>
