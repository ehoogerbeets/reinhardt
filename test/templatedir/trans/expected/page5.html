
<div>
   Untranslated string with Ausgangssatz. in the middle.
</div>
